packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -67,10 +67,15 @@ class WebpackCLI {
     }
 
     // For babel/typescript
-    if (result.default) {
+    if (result && result.default) {
       result = result.default;
     }
 
+    // Treat undefined configuration for zero-config
+    if (!result) {
+      result = {};
+    }
+
     return result;
```

Comment:
Can be simplified by `return result || {}`;

Reply:
And lets add case where `module.exports.default = undefined`

Reply:
Okay, will do.